### PR TITLE
Reproduce results

### PR DESCRIPTION
**Pull request by de-code**
Some changes to reproduce the results locally. I removed the output for now as it may not match exactly your results yet. I will probably add them back in at the end. The main missing part is `analyse_papers.ipynb` as df1000 isn't defined.

(Also the scatter plots in the criticism analysis came out blank, I think I fixed it but not sure if the TSNE plot is right)

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Slightly related: does this project need a build somewhere?

**Comment by de-code** [comment]
That's probably not necessary at the moment. Perhaps if, when we take it further in the future?

**Comment by giorgiosironi** [comment]
:+1:

**Comment by de-code** [comment]
I've updated the analyse_papers notebook and added the output for most notebooks. Could you please check whether that looks okay and we could merge it?

File: front.ipynb [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+   "source": [
+    "# We set a threshold of minimum of a minimun number of papers found citing the DOI for\n",
+    "# the cited paper to be considered:\n",
+    "minimun_number_citations = 10\n",
+    "%store minimun_number_citations \n",
+    "\n",
+    "# We also willplot the analysis for a particular cited paper from the list of DOIs\n",
+    "# Set DOI_cited_list_number to a value out of the list lenght range to skip (e.g. -1):\n",
+    "DOI_cited_list_index = 0\n",
+    "%store DOI_cited_list_index \n",
```
Comment (de-code) [inline front.ipynb:279]: Name changed

File: front.ipynb [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+    }
+   ],
+   "source": [
+    "# We set a threshold of minimum of a minimun number of papers found citing the DOI for\n",
+    "# the cited paper to be considered:\n",
+    "minimun_number_citations = 10\n",
+    "%store minimun_number_citations \n",
+    "\n",
+    "# We also willplot the analysis for a particular cited paper from the list of DOIs\n",
+    "# Set DOI_cited_list_number to a value out of the list lenght range to skip (e.g. -1):\n",
```
Comment (de-code) [inline front.ipynb:277]: Minor typo "length"

File: front.ipynb [hunk 3]
```diff
@@ -0,0 +1,500 @@
...
+   ],
+   "source": [
+    "\"\"\"\n",
+    "Select file containing \n",
+    "dataframe['cited_DOI', 'citing_DOI', 'reference_id', 'introduction_found', 'cited_in_introduction', 'maintext_found', 'cited_in_maintext', 'discussion_found', 'cited_in_discussion', 'conclusions_found', 'cited_in_conclusions', 'sentence_citing_intro', 'sentence_citing_maintext', 'sentence_citing_discussion', 'sentence_citing_conclusions']\n",
+    "and a particular 'cited_DOI' to analyse.\n",
+    "\"\"\"\n",
+    "\n",
+    "cited_DOI_to_analyse = \"10.1126/science.1179052\"\n",
+    "file_df = 'DOI_cited_science_1179052_retracted'\n",
```
Comment (de-code) [inline front.ipynb:314]: Minor naming - I'd probably use the `_df` suffix for DataFrames only.

**Pull request merged**